README.md

```diff
@@ -1,54 +1,50 @@
 NioCommand
-===========
-Command a nio service or block in a running nio service.
+==========
 
-One request is made for every input signal. For each successful request, an output signal is created with the json response (if response was json) or a signal with a `resp` attribute.
 
 Properties
---------------
--   **service**: Name of service to command.
--   **block**: Name of block to command. If blank, then the service will be commanded.
--   **command**: Name of service or block command.
--   **params**: Key/Value pairs to pass as parameters to the command.
--   **host**: Location of nio instance.
--   **port**: Port of nio instance.
--   **security_method**: HTTP Method (ex. NONE, BASIC (default), OAUTH).
--   **basic_auth_creds**: When using Basic Authentication, enter the username and password.
--   **private_key**: When using OAuth, enter the location of the Service Account JSON key file. The path is relative to the root of the nio project. (ex. `etc/private_key.json`)
+----------
+- **basic_auth_creds**: When using Basic Authentication, enter the username and password.
+- **block_name**: Name of block to command. If blank, then the service will be commanded.
+- **command_name**: Name of service or block command.
+- **host**: Location of nio instance.
+- **key_config_file**: When using OAuth, enter the location of the Service Account JSON key file. The path is relative to the root of the nio project. (ex. `etc/private_key.json`)
+- **params**: Key/Value pairs to pass as parameters to the command.
+- **port**: Port of nio instance.
+- **reauth_interval**: How frequently to re-authenticate OAuth login.
+- **security_method**: HTTP Method (ex. NONE, BASIC (default), OAUTH).
+- **service_name**: Name of service to command.
+
+Inputs
+------
+- **default**: Any list of signals.
+
+Outputs
+-------
+- **default**: One output signal will be created for each command (and therefore one for each input signal).  
+If the command response is json, then the Signal representation of the json will be emitted.  
+If the command response is not json, then the Signals will have an attribtue `resp` with the text representaion of the response as the value.
+
+Commands
+--------
 
 Dependencies
-----------------
+------------
 - [oauth2client](https://github.com/google/oauth2client)
 - [pycrypto](https://pypi.python.org/pypi/pycrypto) or [pyopenssl](https://pypi.python.org/pypi/pyOpenSSL)
```

Comment:
In the requirements.txt file, it lists `requests` and `oauth2client`, maybe these are out of date?